jobs/aws/eks-anywhere-build-tooling/kind-presubmits.yaml

```diff
@@ -16,7 +16,7 @@ presubmits:
   aws/eks-anywhere-build-tooling:
   - name: kind-tooling-presubmit
     always_run: false
-    run_if_changed: "EKS_DISTRO_BASE_TAG_FILE|EKSD_LATEST_RELEASES|^build/lib/.*|Common.mk|projects/kubernetes-sigs/kind/.*"
+    run_if_changed: "EKS_DISTRO_MINIMAL_BASE_KIND_TAG_FILE|EKSD_LATEST_RELEASES|^build/lib/.*|Common.mk|projects/kubernetes-sigs/kind/.*"
```

Comment:
Went ahead and changed this since it bases off a different base now. Not sure if there is somewhere on the cdk side we should change?

Reply:
nah we don't need CDK changes for this.